### PR TITLE
Looooooong messages

### PR DESCRIPTION
**Pull request by rowanseymour**

### CODE REVIEW

File: temba/sql/current_indexes.sql [hunk 1]
```diff
@@ -115,6 +115,6 @@ CREATE INDEX CONCURRENTLY values_value_field_decimal_value_not_null
 ON values_value(contact_field_id, decimal_value)
 WHERE contact_field_id IS NOT NULL AND decimal_value IS NOT NULL;
 
-CREATE INDEX CONCURRENTLY values_value_field_string_value_concat
-ON values_value((contact_field_id || '|' || UPPER(string_value)));
+CREATE INDEX CONCURRENTLY values_value_field_string_value_concat_new
+ON values_value((contact_field_id || '|' || UPPER(SUBSTRING(string_value, 1, 32))));
```
Comment (rowanseymour) [inline temba/sql/current_indexes.sql]: Forgot to regenerate this file when I last changed this index

**Comment by rowanseymour** [comment]
@nicpottier I was thinking because 1) it's likely not need now and 2) for a charfield it's not actually configurable because an implementer would have to generate migrations

But if I can convert `FlowPathRecentMessage.text` to a TextField then it would be - so I'll try that and put the settings back in.

**Comment by nicpottier** [comment]
Since it doesn't actually change anything on the DB you don't actually NEED to create the migrations, just Django bitches that your models are out of date. (this is the state for Resist)

**Comment by rowanseymour** [comment]
That's true for all the textfields but not the CharField on FlowPathRecentMessage. But if I make described fix then Resist will stop complaining because everything will be unlimited TextFields.

**Comment by nicpottier** [comment]
Nice, let's do that then!

**Pull request merged**